openedx/core/djangoapps/appsembler/tahoe_idp/helpers.py

```diff
@@ -185,3 +193,61 @@ def is_studio_login_form_overridden():
     if settings.FEATURES.get('TAHOE_IDP_STUDIO_LOGIN_FORM_OVERRIDE', None):
         return True
     return False
+
+
+def extract_course_id_from_url(url):
```

Comment:
That would be an easier task to do.

```suggestion
def extract_org_short_name_from_url(url):
```